app/controllers/coronavirus_landing_page_controller.rb

```diff
@@ -4,12 +4,38 @@ class CoronavirusLandingPageController < ApplicationController
   def show
     breadcrumbs = [{ title: t("shared.breadcrumbs_home"), url: "/", is_page_parent: true }]
 
-    render "show",
-           locals: {
-             breadcrumbs:,
-             details: presenter,
-             special_announcement:,
-           }
+    ab_test = GovukAbTesting::AbTest.new(
+      "UxmPracticeTest",
+      dimension: 68,
+      allowed_variants: %w[A B Z],
+      control_variant: "Z",
+    )
+
+    @requested_variant = ab_test.requested_variant(request.headers)
+    @requested_variant.configure_response(response)
+
+    if @requested_variant.variant?("A")
+      render "one_line_break",
```

Comment:
you don't necessarily have to render an entirely different view if you don't want to, though it's a bit subjective. (Can just have a tiny bit of switching logic in a view if it's small enough) I guess your real test will be a more significant change than a couple of line breaks though!
One risk about entirely different views is that someone else tweaks styles or something for the default view and don't know enough to update the others too. Hopefully that's not an issue here though.